### DISCUSSION TITLE
Fail if env is not correct

### DISCUSSION THREAD

**Discussion by zRelux** [goatandsheep/react-native-dotenv#145] [Ideas]
I think it would be a neat feature to the have the babel fail the `.env` does not match a js object. the idea would be to replicate this package https://www.npmjs.com/package/envalid.

Any suggestions?

**Comment by goatandsheep** [comment]
I like this idea a lot, but does it need to be built into `react-native-dotenv`? Like are you able to create a test script that validates your `.env` files and run a command in your app that validates? If so, I can add it as a suggestion of best practices, but probably better to keep it out of this library. Otherwise, if you have some time, feel free to open a ticket + PR!

**Comment by goatandsheep** [comment]
Also, have you looked into alternatives? Personally I found [`dotenv-extended`](https://www.npmjs.com/package/dotenv-extended) easier to wrap my head around because it uses `.schema` files

**Comment by goatandsheep** [comment]
I think we can add a fail to the verbose option
